### ISSUE TITLE
Signature for group method may be incorrect?

### ISSUE THREAD

**Issue by jeking3** [sbdchd/celery-types#46]
I am creating a group using a generator of task signatures:

```
    job = group(
        resolve_value.s(value.id, as_of, http_meta)
        for value, as_of in work.items()
    )
```

This is valid according to the celery source comments:

https://github.com/celery/celery/blob/master/celery/canvas.py#L1009-L1012

>     Note:
>         If only one argument is passed, and that argument is an iterable
>         then that'll be used as the list of tasks instead: this
>         allows us to use ``group`` with generator expressions.

However mypy is complaining:

> value_query.py:150: error: Argument 1 to "group" has incompatible type "Generator[Signature, None, None]"; expected "Signature"

All my tests pass however, so it appears the type declaration is not complete here.
Here's how Visual Code sees it:

![image](https://user-images.githubusercontent.com/11771245/140618422-5d5027ff-6686-4707-844e-0d60a8af1dbd.png)

**Comment by sbdchd** [comment]
Should be fixed in 0.9.2

Thanks for the report!

**Comment by jeking3** [comment]
Wow.  You fixed that so fast, it caused the type: ignore workaround in my pull request to fail!  Thanks!
